### ISSUE TITLE
Add option to group by Epics

### ISSUE THREAD

**Issue by cschroeder** [Widen/jira-changelog#6]
Currently, all story and bugs issues are output with no mention of any JIRA epic issues they belong to.  An option to group all issues by epics would be helpful, like the below example:

EPIC PROJ-0001 - {Epic Summary}
Story PROJ-1002 - {Story Summary}
Story PROJ-1003 - {Story Summary}

EPIC PROJ-0010 - {Epic Summary}
Story PROJ-2002 - {Story Summary}

MISC {No Epic}
Story PROJ-1111 - {Story Summary}
Story PROJ-2222 - {Story Summary}

**Comment by rnicholus** [comment]
@cschroeder How would you expect/like epics and their child issues to be grouped together when output as markdown? Perhaps as list items? What about issues without epic parents?

**Comment by cschroeder** [comment]
@rnicholus Yes, JIRA stories presented as list items beneath the Epic sounds reasonable.  And for stories with no epic, I was thinking the heading would read MISC or No Epic like in the example above...
